gems/smithy/spec/spec_helper.rb

```diff
@@ -73,6 +60,38 @@ def create_plan(modules, model, type, options)
       }
       Smithy::Plan.new(model, type, plan_options)
     end
+
+    ## end temporary
+
+    def generate_client_gem(options = {})
+      plan = ClientHelper.generate(:client, options)
+      RbsSpyTest.setup(modules, sdk_dir) if ENV.fetch('SMITHY_RUBY_RBS_TEST', false)
+      $LOAD_PATH << "#{plan.destination_root}/lib"
+      require plan.gem_name
+      RbsSpyTest.setup(plan.module_name, plan.destination_root) if ENV['SMITHY_RUBY_RBS_TEST']
+      plan
+    rescue LoadError => e
+      puts "Error loading gem: #{plan.gem_name}"
+      raise e
+    end
+
+    def generate_client_from_source(options = {})
```

Comment:
from source I still find confusing  maybe `generate_in_memory_client` or something like that.